### ISSUE TITLE
change the font size 

### ISSUE THREAD

**Issue by greweln** [Shizcow/dmenu-rs#58]
I need a bit of a guidance about how the font size could be increased  (since there's no flag for it).

Thanks

**Comment by Shizcow** [comment]
When using the `--fn` flag, specify fonts in [font name syntax](https://www.freedesktop.org/software/fontconfig/fontconfig-user.html). To specify the font size, put it after a dash after the font name. For example, if you want to use the `Times` font at `12` point, launch dmenu via `dmenu --fn 'Times-12'`. Now, arbitrary font sizes aren't always available for each font. The recommended method to find a good font size is to just try several and see what works / what looks the best.

TLDR: Specify the font point after the font name as `--fn 'FontName-FontSize'`.

**Comment by greweln** [comment]
perfect!  thank you
